pom.xml

```diff
@@ -57,6 +71,11 @@
                 <artifactId>graphql-java</artifactId>
                 <version>${graphql-java.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.kafka</groupId>
+                <artifactId>kafka-clients</artifactId>
+                <version>${kafka.version}</version>
```

Comment:
as mentioned more times we should avoid this in order to bring the dependency with Vert.x Kafka client.